Userland/Libraries/LibGfx/Painter.cpp

```diff
@@ -671,6 +671,12 @@ void Painter::draw_bitmap(IntPoint const& p, GlyphBitmap const& bitmap, Color co
     }
 }
 
+void Painter::draw_triangle(IntPoint offset, AK::Vector<IntPoint> const& control_points, Color color)
+{
+    VERIFY(control_points.size() >= 3);
```

Comment:
It feels like this should be `== 3` instead. What does a triangle with > 3 corners mean?